src/DependencyInjection/LeagueOAuth2ServerExtension.php

```diff
@@ -241,6 +265,18 @@ private function configurePersistence(LoaderInterface $loader, ContainerBuilder
         }
     }
 
+    private function isDoctrinePersistenceEnabled(array $config): bool
```

Comment:
This method looks like it's dead code that can be removed.

Reply:
I'll double check why I need this in the first place 🤔

Reply:
updated